WifiManager/Wifi.json

```diff
@@ -116,7 +123,11 @@
             }
         },
         "connect":{
-            "summary": "Attempts to connect to the specified SSID with the given passphrase. Passphrase can be `null` when the network security is `NONE`. When called with no arguments, this method attempts to connect to the saved SSID and password. See `saveSSID`.",
+            "summary": "Attempts to connect to the specified SSID with the given passphrase. Passphrase can be `null` when the network security is `NONE`. When called with no arguments, this method attempts to connect to the saved SSID and password. See `saveSSID`. Triggers `onWIFIStateChanged` and `onError` event.\n \n### Events \n| Event | Description | \n| :----------- | :----------- | \n| `WiFiState: CONNECTING or CONNECTED` | Triggers `onWIFIStateChanged` event when Wifi state changes to CONNECTING, CONNECTED. | \n| `onError` | Triggers `onError` event, if the requested SSID connection fails.|",
```

Comment:
Do not add like this
`WiFiState: CONNECTING or CONNECTED`

Please add separate rows and describe

Reply:
I have added separate for each Wi-Fi state.